src/pokedex.c

```diff
@@ -2347,6 +2347,17 @@ static void PrintMonDexNumAndName(u8 windowId, u8 fontId, const u8 *str, u8 left
     AddTextPrinterParameterized4(windowId, fontId, left * 8, (top * 8) + 1, 0, 0, color, TEXT_SKIP_DRAW, str);
 }
 
+static void PrintMonName(u8 windowId, u8 fontId, const u8 *str, u8 left, u8 top)
```

Comment:
I don't see the harm of keeping these in the same function to avoid duplication of code. Number will very likely never be narrowed, so it shouldn't be affected by this.

Reply:
I think I prefer making it clear which strings are intended to be affected by narrowing. What if I cleaned them up into:
```c
    static const u8 color[3] = { TEXT_COLOR_TRANSPARENT, TEXT_DYNAMIC_COLOR_6, TEXT_COLOR_LIGHT_GRAY };
    fontId = GetFontIdToFit(str, fontId, 0, 50);
    AddTextPrinterParameterized4(windowId, fontId, left * 8, (top * 8) + 1, 0, 0, color, TEXT_SKIP_DRAW, str);
```
And that, but without the middle line. For me personally, that doesn't feel like duplication at all, it's just two similar lines which also appear all over the place in the rest of the code, because it takes two lines to set up a color array and use that to draw text.

Reply:
Gotcha 👍